sirepo/raydata/analysis_driver/__init__.py

```diff
@@ -16,6 +17,10 @@
 
 _ANALYSIS_DRIVERS = PKDict()
 
+_CONDA_PREFIX = "/home/vagrant/miniconda"
```

Comment:
This should be config. There is a non-zero chance we will end up mounting their conda prefix and we shouldn't have paths like /home/vagrant hardcoded.

Reply:
7407ed0e7849051a42a067015819229cd53c406b